src/components/SwiperCarousel.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+// import Swiper core and required modules
+import { Navigation, Scrollbar, A11y, EffectCoverflow } from 'swiper';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+// Import Swiper styles
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/scrollbar';
+import 'swiper/css/effect-coverflow';
+
+const data = [
+  { cover: require('@site/static/img/homepage-carousel/1.png').default, title: 'Preview 1' },
+  { cover: require('@site/static/img/homepage-carousel/2.png').default, title: 'Preview 2' },
+  { cover: require('@site/static/img/homepage-carousel/3.png').default, title: 'Preview 3' },
+  { cover: require('@site/static/img/homepage-carousel/4.png').default, title: 'Preview 4' },
+  { cover: require('@site/static/img/homepage-carousel/5.png').default, title: 'Preview 5' },
+  { cover: require('@site/static/img/homepage-carousel/6.png').default, title: 'Preview 6' },
+  { cover: require('@site/static/img/homepage-carousel/7.png').default, title: 'Preview 7' },
+];
+
+// eslint-disable-next-line react/display-name
+export default () => {
+  const slides = data.map((item, index) => (
+    <SwiperSlide key={index}>
+      <img src={item.cover} alt={item.title} />
+    </SwiperSlide>
+  ));
+
+  return (
+    <Swiper
+      // install Swiper modules
+      modules={[Navigation, Scrollbar, A11y, EffectCoverflow]}
+      spaceBetween={-200}
+      slidesPerView={3}
+      navigation
+      loop
+      simulateTouch
+      // onSwiper={(swiper) => console.log(swiper)}
+      centeredSlides={true}
+      // onSlideChange={() => console.log('slide change')}
```

Comment:
I'm going to let these slide (get it 😉) so we remember that they exist in case we might want to use them.